GI01SUMO/GI01SUMO_R00007_ingest.csv

```diff
@@ -0,0 +1,79 @@
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/CTDMO_sn37-13628/*.hex,GI01SUMO-RII11-02-CTDMOR018,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdmo09/*.DAT,GI01SUMO-RII11-02-CTDMOR019,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/CTDMO_sn37-13629/*.hex,GI01SUMO-RII11-02-CTDMOR019,recovered_inst,Available,
+mi.dataset.driver.ctdmo_ghqr.imodem.ctdmo_ghqr_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdmo10/*.DAT,GI01SUMO-RII11-02-CTDMOR020,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/CTDMO_sn37-13630/*.hex,GI01SUMO-RII11-02-CTDMOR020,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/CTDBP_sn16-50156/SBE*.hex,GI01SUMO-RII11-02-DOSTAD031,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp01/*.DAT,GI01SUMO-RII11-02-DOSTAD031,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/CTDBP_sn16-50157/SBE*.hex,GI01SUMO-RII11-02-DOSTAD032,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp02/*.DAT,GI01SUMO-RII11-02-DOSTAD032,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dosta_abcdjm_ctdbp_p_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/instruments/dcl11/imm/CTDBP_sn16-50158/SBE*.hex,GI01SUMO-RII11-02-DOSTAD033,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp_p.dcl.dosta_abcdjm_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp03/*.DAT,GI01SUMO-RII11-02-DOSTAD033,recovered_host,Available,
+mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp01/*.DAT,GI01SUMO-RII11-02-FLORDG031,recovered_host,Available,
+mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp02/*.DAT,GI01SUMO-RII11-02-FLORDG032,recovered_host,Available,
+mi.dataset.driver.flord_g.ctdbp_p.dcl.flord_g_ctdbp_p_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00007/cg_data/dcl11/imm/ctdbp03/*.DAT,GI01SUMO-RII11-02-FLORDG033,recovered_host,Available,

```

Comment:
Should there be lines for FLORD recovered_instr using flord_g_ctdbp_p_recovered_driver?